Mathlib/Topology/UniformSpace/Closeds.lean

```diff
@@ -244,6 +244,17 @@ theorem isClosedEmbedding_singleton :
 
 end T0Space
 
+instance t0Space : T0Space (Closeds α) := by
```

Comment:
```suggestion
instance : T0Space (Closeds α) := by
```
Usually we go with the auto-generated name. Is there a reason to name manually in this case?